lib/components/Dropdown/index.tsx

```diff
@@ -162,19 +214,36 @@ export const Dropdown = ({
                 `deriv-dropdown__items--${listHeight}`,
                 {
                     'deriv-dropdown__items--full': isFullWidth,
-             }
+                }
             )} {...getMenuProps()}>
                 {isOpen && (
-                    items.map((item, index) => (
+                    <div>
+                    {noResults && <li className="no-results-found">No results found</li>}
```

Comment:
```suggestion
                    {noResults && <li className="no-results-found">{noResultMessage}</li>}
```
can we add a props called `noResultMessage` ? because they want to pass the message

Reply:
I noticed that inside the design file it's like this:
![image](https://github.com/deriv-com/ui/assets/100833613/5499a613-6cf0-4e3e-81db-09b58be8f4a1)

but you need to modify it and include the query inside the message like the design

so maybe we don't need to pass any message from outside 🤔